src/bindings/python/wheel/setup.py

```diff
@@ -290,6 +290,11 @@ def configure(self, install_cfg):
                 install_dir = os.path.join(install_prefix, install_dir)
             # set rpath if applicable
             if sys.platform != "win32" and comp_data.get("rpath"):
+                # after tbb libraries on mac arm64 are signed, setting rpath for them will report error:
+                # LC_SEGMENT_64 command 3 fileoff field plus filesize field extends past the end of the file
+                if comp == "tbb_libs" and ARCH == "arm64" and sys.platform == "darwin":
+                    continue
```

Comment:
I suppose the WA could be: compile with hwloc statically and totally remove a need to set RPATH for TBB libraries

Without RPATH, TBB cannot load TBBBind (because TBBBind cannot load hwloc library) and understand information about P / E cores. So, it may affect latency and tput, right? @wangleis

Reply:
There are no tbbbind library for MACOS

Reply:
@ilya-lavrenov So far there is no latency and tput on MACOS yet. We will investigate other method to identify core type when we enable latency and tput on MACOS.